src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/MutationParameter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * pottery-backend-interface - Backend API for testing programming exercises
+ * Copyright © 2015-2018 BlueOptima Limited, Andrew Rice (acr31@cam.ac.uk)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package uk.ac.cam.cl.dtg.teaching.pottery.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
+public class MutationParameter {
+  @ApiModelProperty("The name of this parameter.")
+  private final String name;
+
+  @ApiModelProperty("The values that are drawn for this parameter.")
+  private final String value;
```

Comment:
ugh. confusion (at my end). if these are values why is this a singleton string?

Reply:
This string is the name of the value set to draw a value from. Better terminology would be welcome here...

Reply:
valuesKey? valueSetName?

Reply:
I like `valueSetName`, will change it.